scripts/normalizeMarkdown.js

```diff
@@ -0,0 +1,387 @@
+// run: node scripts/normalizeMarkdown.js "product_docs/**/*.mdx"
```

Comment:
I think it would be nice if these scripts were in a `scripts/normalization` directory or something similar, what do you think?

Reply:
We should also extend the precommit hook in package.json to cover these files